src/problem-analysis/state-of-the-art/chess-com.tex

```diff
@@ -1,4 +1,33 @@
 % textidote: ignore begin
 \subsection{Chess.com}\label{subsec:chess-com}
+
+\subsection{What is Chess.com?}\label{subsec:what-is-chess-com}
```

Comment:
Maybe delete this line and just make the "Chess.com" the subsection title. I dont know though, just a thought